### ISSUE TITLE
Only minify on live server

### ISSUE THREAD

**Issue by musdy** [HTMLMin/Laravel-HTMLMin#99]
Is there a way to stop minification on local and staging?

**Comment by Tickthokk** [comment]
Not the maintainer, but I think you could probably just tie it to APP_DEBUG in the `htmlmin.php` config file.

```php
    'blade' => ! env('APP_DEBUG'),
```

instead of 

```php
    'blade' => true,
```

**Comment by test82987** [comment]
no. APP_DEBUG is a harmful thing to enable on life when spesific property can disable the package but keep the app-debug secure.
